docs/source/deprecated/api.rst

```diff
@@ -725,20 +726,22 @@ Property                                     Type     Units
 ``tankVolume``                               float    gal                                         no
 ``dhwLoadFraction``                          float    fraction                    0-1             yes                             sum of dhwLoadFraction must equal 1
 ``heatCapacityBtuPerHour``                   float    Btu/hr                      >0              no                              autosized by modeling engine if undefined
-``energyFactor`` or ``uniformEnergyFactor``  float    fraction                    <1              no        BSA
+``efficiency``                               float    fraction                    >0              no        BSA
+``efficiencyUnits``                          string                               see [#]_        no        fraction
```

Comment:
@kellyatradiantlabs  Should you need to remove `fraction` from the Default column associated to `efficiencyUnits`?

Reply:
Note this is on the deprecated version of this page. I updated the current version though! See request_structure.rst